xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -1322,24 +1322,6 @@ edit.defaultEditor.org.xwiki.rendering.block.XDOM#wysiwyg=$xwikiPropertiesDefaul
 #-------------------------------------------------------------------------------------
 # Event Stream
 #-------------------------------------------------------------------------------------
-
-#-# [Since 11.1RC1]
```

Comment:
I think you should keep that, since I assume it also applies to the property you kept.

Reply:
I don't think so, the property I kept is older than that AFAIR, but I'll check.